edgecontext.thrift

```diff
@@ -124,7 +123,8 @@ interface provided by baseplate.
 */
 struct Locale {
     /** IETF language code representing the client locale preferences.
-    Can be either {lang} or {lang}_{region} format. e.g. en, en_US
+    Preferably in BCP-47 format ({lang} or {lang}-{region}), 
+    but underscore separated locales also valid ({lang}_{region})
```

Comment:
can we make the two comments consistent? this is the only place the type is used, so there's really not a reason the comments should say different things. 

we probably can remove the whole comment about the format here because the format is already explained at the type.